fasm2bels/lib/interchange_capnp.py

```diff
@@ -522,6 +531,8 @@ def finish_encode(self):
             for stub_obj, stub in zip(net.stubs, stubs):
                 stub.output_interchange(stub_obj, self.string_id)
 
+            net.type = net_to_type.get(net_name, 'signal')
```

Comment:
Where is net.type used, and why should it default to signal?

Reply:
It's part of the FPGA interchange output.  I'm working on moving the FPGA interchange stuff to its own library, and it has better docs.